src/assets/css/namespaces.css

```diff
@@ -0,0 +1 @@
+@namespace svg "http://www.w3.org/2000/svg";
```

Comment:
Why is this needed, out of curiosity? For `:not(svg)` to work below?

Reply:
That's right, with `:not(svg|*)` I am excluding all elements of svg namespace. Without this exclusion, svg icons stop working.